docs/environment-vars.md

```diff
@@ -0,0 +1,16 @@
+## Environment variables
```

Comment:
This file is all about describing the docker image that this repo is designed to build. It is meant to be read by people using the image rather than by people adjusting how the image its self is built.

So to me, it could make sense to leave this in the top level README.md. Perhaps at the very top and then put the instructions for how the image is build can be put beneath that